src/controller/user.ts

```diff
@@ -0,0 +1,214 @@
+import _ from 'lodash'
+import { BaseContext } from 'koa'
+import { getManager, Repository, Not, Equal } from 'typeorm'
+import { request, summary, path, body, responses, tagsAll, description } from 'koa-swagger-decorator'
+import { User, userSchema } from '../entity/user'
+import { validate, validatePassword, requiredProperties } from '../utils/validate'
+import { safeCall, response } from '../utils/helpers'
+
+@tagsAll(['User'])
+export default class UserController {
+    @request('get', '/users')
+    @summary('Find all users')
+    @responses({
+        200: { description: 'success' },
+        400: { description: 'error' },
+        401: { description: 'token authorization error' },
```

Comment:
when will this API return 400, I cannot find it in implementation code and test code.

Reply:
No it does not return `400`. `400` should not be among the responses for this endpoint in Swagger docs. An error on my part.